tutorials/overleaf/index.mdx

```diff
@@ -0,0 +1,203 @@
+
+If the previous command runs overleaf correctly, it is time to create a systemd daemon service.
+
+1. Create and open the file `/etc/systemd/system/overleaf.service` in a text editor of your choice and past the following content into it:
+    ```
+    [Unit]
+    Description=Overleaf Community Edition
+
+    [Service]
+    User=root
+    WorkingDirectory=/opt/overleaf-toolkit
+    ExecStart=/opt/overleaf-toolkit/bin/up
+    Restart=always
+    RestartSec=10
+
+    [Install]
+    WantedBy=multi-user.target
+
+    [Unit]
+    Description=Overleaf Community Edition
+    
+    [Service]
+    User=root
+    WorkingDirectory=/opt/overleaf-toolkit
+    ExecStart=/opt/overleaf-toolkit/bin/up
+    Restart=always
+    RestartSec=10
+    
+    [Install]
+    WantedBy=multi-user.target
+    ```
+2. Save the file and exit the text editor.
+3. Reload systemd by runnning the following command:
+    ```
+    systemctl daemon-reload
+    ```
+4. Enable the `overleaf.service` to be started on reboot
+    ```
+    systemctl enable overleaf.service
+    ```
+5. Start the Overleaf service:
+    ```
+    systemctl start overleaf.service
+    ```
+6. Check the status of the Overleaf service:
+    ```
+    systemctl status overleaf.service
+    ```
+
+## Installing LaTeX packages
+
+The LaTeX packages can now be installed, using TeX Live.
+1. Run the following command to launch the Texlive manager update.
+    ```
+    docker exec Overleaf Community Edition tlmgr update --self
+    ```
+2. Run the following command to lauch the full `texlive` installation
+    ```
+    docker exec Overleaf Community Edition tlmgr install scheme-full
+    ```
+    <Message type="note">
+      The installation of `texlive` may take a while.
+    </Message>
+
+## Configuring Nginx
+
+This tutorial uses [NGINX](https://nginx.org/en/) with [Let's Encrypt](https://letsencrypt.org/) and [CertBot](https://certbot.eff.org/) as an SSL enabled reverse proxy to secure the connection between the client and our server.
+
+1. Install Certbot on the server:
+    ```
+    apt install python3-certbot-nginx
+    ```
+2. Generate the certificate, by running the following command:
+    ```
+    sudo certbot --nginx certonly
+    ```
+
+    SSL certificates issued by Let's Encrypt are valid for 90 days. Certbot comes with the option to automatically renew the certificate.
+3. Configure a cron job that launches Certbot daily to check the status of the certificate. Open the crontab by typing:
+    ```
+    crontab -e
+    ```
+4. Add the following line which will stop Nginx, renew the certificate and restart the web server daily at 4:30 in the morning:
+    ```
+    30 4 * * * /opt/letsencrypt/certbot-auto renew --pre-hook "service nginx stop" --post-hook "service nginx start"
+    ```
+5.  Edit the file `/etc/nginx/sites-enabled/default` so that it looks like the following example, to configure Nginx in a way, that it will serve our Overleaf Community Edition site as default site:
```

Comment:
```suggestion
5.  Edit the `/etc/nginx/sites-enabled/default` file so that it looks like the example below. This allows you to configure Nginx in a way that will serve our Overleaf Community Edition site as default site:
```